contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -289,8 +300,8 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
     // No lower bound on the requested gas limit. A user could request 0
     // and they would simply be billed for the gas and computation.
-    if (billing.callbackGasLimit > s_config.maxCallbackGasLimit) {
-      revert GasLimitTooBig(billing.callbackGasLimit, s_config.maxCallbackGasLimit);
+    if (billing.callbackGasLimit > _getMaxCallbackGasLimit(billing.flags)) {
```

Comment:
Is there a reason that `_getMaxCallbackGasLimit(billing.flags)` is called twice instead of setting it to a variable?

Reply:
My omission, will fix.

Reply:
My bad, actually call stack :/